### ISSUE TITLE
‘javadoc’ task fails on master

### ISSUE THREAD

**Issue by jcxldn** [SolteraGG/StickyAPI#62]
Error log:

```
StickyAPI/src/main/java/com/dumbdogdiner/stickyapi/common/util/FieldUtil.java:8: error: cannot find symbol
import java.lang.invoke.VarHandle;
                       ^
  symbol:   class VarHandle
  location: package java.lang.invoke
StickyAPI/src/main/java/com/dumbdogdiner/stickyapi/common/util/FieldUtil.java:20: error: cannot find symbol
    private static final VarHandle MODIFIERS;
```

**Comment by ZachyFoxx** [comment]
First p0 bug! :tada:

**Comment by aakatz3** [comment]
Similar thing happens (I think??) for GUI#get<something>

**Comment by jcxldn** [comment]
output running in the build container (locally) - `./gradlew javadoc`

```
/stickyapi/build/delombok/main/com/dumbdogdiner/stickyapi/bukkit/command/ExitCode.java:12: error: unexpected content
 * @see {@link com.dumbdogdiner.stickyapi.common.command.ExitCode}
   ^
/stickyapi/build/delombok/main/com/dumbdogdiner/stickyapi/common/command/CommandBuilder.java:38: error: reference not found
     * @return {@link CommandBuilderBase}
```

**Comment by aakatz3** [comment]
`@see {@link blah}` is invalid  
the not founds are the real issue, need to look into a few things furthur.

**Comment by aakatz3** [comment]
Fixed in #56

**Comment by jcxldn** [comment]
Fixed in master.
